pandas/tests/arithmetic/test_period.py

```diff
@@ -608,8 +608,9 @@ def test_pi_add_offset_n_gt1_not_divisible(self, box_with_array):
         result = pi + to_offset('3M')
         tm.assert_equal(result, expected)
 
-        result = to_offset('3M') + pi
-        tm.assert_equal(result, expected)
+        # This test is broken
```

Comment:
Why is this test broken?

Nonetheless, we'd prefer to xfail if it's supposed to fail instead of commenting this out.

Reply:
@mroeschke : Perhaps it's lag, but you're repeating my comments from above.